src/js/player.js

```diff
@@ -975,7 +975,10 @@ class Player extends Component {
    *
    * @method handleTechFullscreenChange
    */
-  handleTechFullscreenChange() {
+  handleTechFullscreenChange(event, data) {
+    if (data && data === 'not-fullscreen') {
```

Comment:
Given the conversation below, just a reminder that this should become `data && data.fullscreen === false` (assume we  want to check for an explicit `false` here and not just undefined).